packages/react/src/components/TextInput/next/TextInput.stories.js

```diff
@@ -61,18 +62,21 @@ export const WithLayer = () => {
         type="text"
         labelText="First layer"
         helperText="Optional help text"
+        id="text-input-1"
       />
       <Layer>
         <TextInput
           type="text"
           labelText="Second layer"
           helperText="Optional help text"
+          id="text-input-2"
         />
         <Layer>
           <TextInput
             type="text"
             labelText="Third layer"
             helperText="Optional help text"
+            id="text-input-3"
```

Comment:
My IDE was saying a warning that the id was needed as a required element so I just added it